app/commands/v2/put_content.rb

```diff
@@ -98,6 +99,12 @@ def create_links(edition)
         end
       end
 
+      def fetch_embedded_content(edition)
+        return {} if edition[:details]["body"].nil?
+
+        ContentEmbedService.new(edition[:details]["body"]).fetch_links
```

Comment:
I think you might need the same `body.is_a?(Array)` check as you have in DetailsPresenter here (or maybe pull it down into ContentEmbedService)